scalability_and_performance/ztp-zero-touch-provisioning.adoc

```diff
@@ -26,3 +25,46 @@ include::modules/ztp-low-latency-for-distributed-units-dus.adoc[leveloffset=+1]
 include::modules/ztp-single-node-clusters.adoc[leveloffset=+1]
 
 include::modules/ztp-the-gitops-approach.adoc[leveloffset=+1]
+
+//planning and prereqs
+
+include::modules/ztp-acm-preparing-to-install-disconnected-acm.adoc[leveloffset=+1]
+
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+Deploying a disconnected registry host based on the `docker.io/library/registry:2` API for {product-title} is not officially supported by Red Hat. You can create a mirror host based on the `docker.io/library/registry:2` API with the following unsupported procedure.
+====
+
+include::modules/installation-creating-mirror-registry.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+You must have access to the internet to obtain the necessary container images.
+In this procedure, you place the mirror registry on a mirror host
+that has access to both your network and the Internet. If you do not have access
+to a mirror host, use the disconnected procedure to copy images to a device you
+can move across network boundaries with.
+====
+
+[id="installing-preparing-mirror_{context}"]
+== Preparing the mirror host
+
+Before you perform the mirror procedure, you must prepare the mirror host to retrieve the {product-title} release images.
+
+include::modules/cli-installing-cli.adoc[leveloffset=+2]
+
+include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
+
+include::modules/installation-mirror-repository.adoc[leveloffset=+1]
+
+include::modules/ztp-acm-adding-images-to-mirror-registry.adoc[leveloffset=+1]
+
+include::modules/ztp-acm-installing-disconnected-rhacm.adoc[leveloffset=+1]
+
+include::modules/ztp-site-planning-for-du-deployments.adoc[leveloffset=+1]
```

Comment:
Are you sure the site planning comes this late in the process? I wasn't expecting another planning topic this deep.